bftengine/src/bftengine/BFTEngine.cpp

```diff
@@ -1,24 +1,27 @@
 // Concord
 //
-// Copyright (c) 2018 VMware, Inc. All Rights Reserved.
+// Copyright (c) 2018-2019 VMware, Inc. All Rights Reserved.
 //
 // This product is licensed to you under the Apache 2.0 license (the "License").
 // You may not use this product except in compliance with the Apache 2.0
 // License.
 //
 // This product may include a number of subcomponents with separate copyright
-// notices and license terms. Your use of these subcomponents is subject to the
-// terms and conditions of the subcomponent's license, as noted in the LICENSE
+// notices and license terms. Your use of these sub-components is subject to the
+// terms and conditions of the sub-component's license, as noted in the LICENSE
```

Comment:
Seems strange to change the copyright message, especially since subcomponent is a valid word, and the same copyright is used across most of our files.